### ISSUE TITLE
First draft

### ISSUE THREAD

**Issue by jensroes** [jensroes/prowrite-mixture-models#5]
[manuscript.pdf](https://github.com/jensroes/prowrite-mixture-models/files/12623817/manuscript.pdf)

I have a rough first version ready. 

There are gaps in the methods (see #4) and I need to go over the discussion again. I'd be happy about general feedback though. Do think the general story is good. Am I missing anything substantial? Is the structure okay (e.g. there is something about comparing complex key combinations in the results which could do into the discussion). Feel free to make suggestions what we could add to the discussion (I feel there is still room). Also do the model descriptions, especially the links to the writing models make sense?

@Mark-Torrance if you have any time at all, probably focus first on the PRB paper first unless you can't wait :)

@RConijn, if you have time for this at the moment, I'd be happy about feedback :)

**Comment by RConijn** [comment]
@jensroes : this took way too long, but here some of my thoughts:

**Third paragraph (aim), p.3/p.4**: The aim comes a bit out of the blue here, I think you would want to already briefly introduce the two theories. 
“we revisit the often reported finding, […] Griffin, 2001 ” This sounds a bit abstract here, without any context yet. Maybe you can move this to later in the introduction (for example to p.5, first para).

**Discussion serial + cascading model, p.4/p.5**: I feel the intro is already quite biased towards the cascading model. If this is intentional, fine, but in that case I think we can be explicit about that as well (in my opinion at least). That is, rather than stating that we compare the two models, you might argue that we aim to find statistical evidence for the cascading model over the serial model. And, in the hypotheses on p.10, we then also want to take position. Otherwise, we need to explain a bit more on the serial model, and extend esp. where you say “this is particularly reflected” in F&H (p.4).

**p.7: “For example, …”**: here I think we could add one sentence explaining what you are exemplifying here (not sure: that you showed two distributions of IKI before, that you can use Bayesian mixture modelling to show a mixture of distributions, or something else?)

**Disadvantages of hesitations, p. 8**: this para feels a bit out of place (maybe delete?)

**p.8 spontaneous text production**: make explicit here that we are analyzing the models on spontaneous text composition (somewhere else you were talking about ‘free text production’, e.g., in the abstract). We need to be consistent there (see also section about data sets p. 18-p.19). I think the Plantra dataset is the most difficult one to categorize I thinkk, it arguably could be considered spontaneous, but not entirely free production… Ah, on p.9 you give a definition: not artificially constrained content generation of linguistic units. Maybe just specifying the definition would help here. Small thing: I’d swap reason 2 and 3, as I see 3 as a stronger argument than 2.

**Statistical models (p.11-p.17)**: I wonder how much explanation is needed here and if we can (and should) assume some basic knowledge. On the other hand I like that the explanations are making it super clear to understand. Not sure😊.Some parts that might be superfluous or could be more concise:  
-	“Regression formulas can […] words and sentences).” (p.12)
-	“In particular, a Guassian […] planning sentences).” (p. 13)

Minor thing: Brackets need to be reversed in equation 4 (p.16). 0 < theta < 1. Also, do we want to add sigma_diff to constraints (sigma_diff > 0, is this correct?) (same comments for equation 5).
I think the first para on p.16 can be written down a bit more clearly, remind me to do this in  a later stage when we’re focusing on small changes etc.

**Data reduction p. 24**: first 2 sentences: move to transition types (some overlap as well). Did you actually have 100 observations for between sentence transitions per participant? I assume not, right? “For the LIFT data aste … our data pool”  move to first para.

**Results, p.25-p.33**: Move all non-results to methods: first para on p. 25. Table 6: align with results in text. Also check overall ordering of results here. E.g., para “Outstanding are … […] other data sets.” Should me moved to part on fluent transition duration (or to discussion). 

"These inconsistencies could, to some extent, be explained ..." (p.30). In this way it sounds really like a discussion, although it adds quite some new results so I feel it's better here in the results section. Maybe it is good to rephrase it a bit. (To control for the use of complex keys.... or something like that). 

**“Further comparisons can be found …” p.31**: what is the goal here? Maybe add some short summary of results(?) otherwise I feel we can skip those comparisons.

**Discussion**: Para “patterns observed in […] before-word locations” – needs some revision to make it more clear which results are connected to which conclusions (happy to help out here).  

**“Fourth, pauses were longer before sentences …”** --> why could the absence of this pattern be specific to a synthesis writing task? Because there is less planning needed in terms of content creation? But in that case you might have expected similar patterns in the PlanTra dataset.

**Simulation approach (p. 36):** this feels a bit more like results again, maybe phrase it similar to the exploration of the SHIFT-key and move it to the results). 

**Not planning but reading (p. 37).** “We removed transitions that terminated in an edititing operation, so […] unlikely to reflect revisions”. I do not completely agree here: in the models of revising (e.g., Flower, 1986; Scardamalia & Bereiter, 1983), they distinguish three processes compare-diagnose-operate, in which ‘operate’ could also mean doing nothing. Also consider internal vs. external revisions, where only the latter will be found in the keystroke log by means of editing operations (although arguably internal revisions could be seen as some planning as well…).  Either way, I think we should be a bit less firm in our conclusions here / make it a bit more compelling. Do we also want to say something about source-interactions here?

**Comment by jensroes** [comment]
Thanks, @RConijn. That's already much more focused on details than I expected at this stage but really helpful. I'll go through those before we have a go at details like wording.

Just two questions about the last comment:

> Not planning but reading (p. 37). “We removed transitions that terminated in an edititing operation, so […] unlikely to reflect revisions”. I do not completely agree here: in the models of revising (e.g., Flower, 1986; Scardamalia & Bereiter, 1983), they distinguish three processes compare-diagnose-operate, in which ‘operate’ could also mean doing nothing. Also consider internal vs. external revisions, where only the latter will be found in the keystroke log by means of editing operations (although arguably internal revisions could be seen as some planning as well…). Either way, I think we should be a bit less firm in our conclusions here / make it a bit more compelling. 

Not sure where to start; I don't understand the problem. I think being firm helps to communicate the point we're making but we can of course add caveats. What is the problem and how could we solve it? Would it be worth adding 1-2 sentence to highlight the caveat and if that's enough, could you give me 2 sentences :)

> Do we also want to say something about source-interactions here?

It does feel relevant. Is it only plantra that involves source interactions? I don't know the literature around writing from source, so I'd welcome help here. First of all, what would you say about sources?

**Comment by RConijn** [comment]
What I tried to say here: it could be that a writer is reading their text to check if it matches their writing goal, [compare] and then decides to not change anything [diagnose: no error], or alternatively, identifies that there is a mismatch, but the writer does not know how to continue, so does not revise [operate: no change]. So that means that there are still pauses that could be related to a revision, which were just not visible eventually in the process. Yet, I feel that this does not debate the proof for the parallel processing - so  I'm actually not sure if we want to mention this / how (curious to see what you and @Mark-Torrance  think). 

For sources, I'd change the following:
Alternatively at least some if not all pauses in the writing process might be the result of reading or looking back into previously
produced text. For two of the datasets (PlanTra and LIFT) which involved writing from sources, we removed the pauses that were made when the sources (rather than the writing document) was in view or 'focus'. So, the pauses could not be related to reading sources. 

(alternatively you could add that to the methods)

_This actually does not work if you have the sources and the writing window side-by-side - but we do not know if participants did this _

**Comment by Mark-Torrance** [comment]
What's the strategy here? I suggest I give some general comments too, you make some macrostructure changes (but don't worry about tidying up the text too much. You then put on Overleaf and I comment / edit in detail?

**Comment by jensroes** [comment]
Mark, yes that sounds good. I wrote all of it in rmarkdown so Google docs
would probably be better.

**Comment by Mark-Torrance** [comment]
## Mark, General comments

I've just read down to end of results (and not thoroughly in some places - so some of the following might just be overhasty reading). I have been quite critical in places. Have have strong positive regard for the paper overall :)

### Introduction
- I like the introduction. There are some bits there that are a bit too close to other SPL2 and the special issue introduction, though. This is tricky - you don't want to have to invent different ways of saying the same thing. And obviously the special issue introduction itself came from work that you were doing on the analysis in this paper, and your contribution to the special issue. Maybe just leave to me to edit this.
- There is nothing in the introduction (I don't think) that talks about the various manipulations in the datasets that you reanalyse. To be honest, I'm not sure how best to handle this. The masked/unmasked thing, in particular is a bit difficult to justify in the context of your hypotheses. I note that Rianne raised issue around discussion of reading during writing. Possibly better just not to go there. Dyslexic/not, L1/L2 and (possibly) pre-post in PLanTra could be justified on the grounds of manipulating likelihood of disruption to cascade. Masked / unmasked is harder, unless you want to make strong points about preventing cuing from existing text (I've not yet read the discussion) - that seems tangential to main purpose. Ah, and I see that you don't actually report findings from the masked conditions. So don't mention in describing the dataset either? Or just mention in a footnote that participants also completed a task in which.... Really not sure that this is important to mention, though. Similarly for genre, topic and pre-post. Mention briefly in text and say that you aggregated across at that point. Don't foreground theses when describing the data. Did you include the masked condition in the models then just omit from the estimates reported in Figure 1? This seems an odd decision. I think masked should probably be omitted from the models.

### Methods
- I know why you've put the statistical models at the top of the methods section. This is your contribution here. But it looks odd to me. I would start with describing the general approach (briefly - which you do). I'd focus a bit more on the data here - just another sentence or two. Then describe the datasets, then data processing (define text edges),  then the details of the statistical models. So follow the structure for a normal empirical paper.
- Can you get most of the information in the dataset (N, language, task, manipulation if any, data collection tool (possibly), reference) into a table? Then anything else just briefly in the text. Ah, and I see that you've done exactly this. There's a lot of repetition in the text. Can you remove this - possibly adding a bit more information to the table if needed. I'll add information from "my" datasets when I do some more detailed editing.
- I agree with Rianne that there's too much detail in the statistical analysis section. A lot of this can be moved to supplementary materials. All that you need to do, I think, is carefully explain the bi-modal models. 
- Table 2 is needed, but is not very clear at the moment. I would just number the models (remove the next two columns), but then expand the description so that there's a reasonable chance of it being understood without the text.
- I'm also not convinced by the model labelling. All of the models apart from the first are Guassian. "Unequal variance" needs to specify where you are allowing variance to vary (i.e. at different edge locations). Most importantly for present purposes unimodal and bi-modal don't tap the most important distinction between these two models - which is one distribution or two distributions. The number of modes is a bi-product of that. Strictly also you can also have single distributions with two modes, too - any that form a U shape). I think just "single distribution" and "mixture model" or "two distribution model" does the right job here.
- I have a more radical suggestion. There is only one model comparison here that dissociates serial and cascaded models - between two-distribution and single-distribution models. Everything else is secondary to that. I can see why you might want to compare (unimodal) normal and log-normal models - just to justify using log-normal in subsequent models (though I kind of still feel that this should be an a priori decision). Unequal variance is definitely an a priori decision, though, isn't it? Why would you even consider assuming equal variances across edge locations? I'm not necessarily saying you should drop these comparisons altogether. Just that the focal comparison should be the main focus that the rest should be demoted to side notes / preliminary comments.

### Results
- I'm struggling with the constrained / unconstrained comparison, for similar reasons. We've talked about this before. I can see why unconstrained will always give a better fit (I think), at least with real data. The strong test of the serial vs. cascaded theory is the comparison between single-distribution unequal variance and constrained bimodal. In all cases this provides better fit. What would happen in you just reported this in Table 5?
- Elpd in table 5 doesn't contribute anything, I think. Just delta elpd?
- Just to repeat myself, Figure 1 worries me. I can't see a rationale, based on the theory that you've developed for allowing the first component to vary across edge location.
- I _think_ Figure 1 would actually be easier to read as a table (much as I like your plots). It might, though, be an idea to include an illustrative distribution plot for just one of the locations (or maybe all three) for just one dataset, like we have in the SPL2 paper. For people not used to thinking about mixture models I think this is really helpful.
- If you present the Figure 1 data as a table you could then combine with Table 6, just adding one column with BFs for the two location comparisons. The actually parameter estimates don't contribute anything to understanding that you can't get for the estimated means and PIs. I would relegate these to appendices.

**Comment by jensroes** [comment]
I'm working through this at the moment, and I think I need some feedback on a few points. All really good point and I've already included some of them:

Re Mark's point 1 above: Manipulations in all datasets would predict interrupts in the process cascade. I agree we shouldn't emphasise this a lot, certainly not in the introduction cause its not important to use. We do want to mention them in the methods and I want to keep them in the analysis and show comparisons in the appendix because this allows a lot of people to relate the results of the mixture models to their own work, I think it can get reviewers exited as there is a lot of information, and, on a lazy note, I really don't think its a good move to run all models again just to remove the manipulations. I can see that removing manipulations would streamline the presentation of this work but I think the side strands are of interest for researchers and makes this paper more relatable.


> There is only one model comparison here that dissociates serial and cascaded models - between two-distribution and single-distribution models. Everything else is secondary to that. [...] Unequal variance is definitely an a priori decision, though, isn't it? Why would you even consider assuming equal variances across edge locations?


You are of course right. The reason why I include the other versions of single distribution models is because they sort of sum up standard practice in the literature, as I understand it, and people feel strongly about single process models and I have heard arguments against lognormal distributions. Also, for some reason, equal variance models are a standard for of modelling, probably because that's how anovas and lms are usually implemented as default. I'm including these models because this is what people are familiar with. This is always why my model description starts with Gaussian models and extends them into mixture models. If my models start with unequal variance models, it makes it harder to people to relate this to their own practice, I think.


> can see why unconstrained will always give a better fit (I think), at least with real data. The strong test of the serial vs. cascaded theory is the comparison between single-distribution unequal variance and constrained bimodal. In all cases this provides better fit. What would happen in you just reported this in Table 5?


I don't understand why unconstrained models would always give better fit. Certainly loo doesn't just show better fit because more parameters are involved and, as I said before, for the total fixation duration I actually got no improvement with unconstrained models. Also the improvement for unconstrained models in keystroke as more than substantial. The rationale that I present for unconstrained models is this

"However, letter bigrams and trigrams may be systematically executed faster than transitions between between space and a letter (REFERENCE?) or complex keystrokes that comprise space and shift-letter combinations for upper case characters before sentences (we will address the latter possibility in the results section). This is because bigrams / trigrams might be stored, retrieved and executed as motor codes but not transitions to a space following a word or between a space or shift key press preceding a letter."

However, I agree that this is a bit posthoc and mostly driven by the better fit for unconstrained models. Therefore I like your idea to focus on the posterior of the constraint model. Obviously this will raise the reviewers' eyebrows and they will wonder why are not focusing on the best fitting model. I'm a bit torn here. So first question here is, does my rationale for unconstrained models above convince you?


> I think Figure 1 would actually be easier to read as a table (much as I like your plots). It might, though, be an idea to include an illustrative distribution plot for just one of the locations (or maybe all three) for just one dataset, like we have in the SPL2 paper. For people not used to thinking about mixture models I think this is really helpful.

There is a Gelman paper that agues how many papers hide patterns in their data by showing tables instead of plots. I actually don't understand how it would be easier to interpret these data as table. Also, rhetorically, figures are better at highlighting important results than tables. To be honest, I don't really find the distribution plots (like in the SPL2 paper) helpful to understand what the mixture model does or what the parameters do. I think your annotate figures (Maria's talk) were better for that. I tried to do something similar with a plot in the noun phrase planning scope paper. 


> If you present the Figure 1 data as a table you could then combine with Table 6, just adding one column with BFs for the two location comparisons. The actually parameter estimates don't contribute anything to understanding that you can't get for the estimated means and PIs. I would relegate these to appendices.

Parameter estimates are important because they tell you something about the magnitude of the difference (which is information you also get from the estimated means), and the uncertainty in the estimate for the difference, which is something you don't necessarily get from estimated means and PIs. I think you're right though, this table can get into the appendix too. I'm not sure if there is something like too many appendices, cause this would be number 10.

**Comment by jensroes** [comment]
> The strong test of the serial vs. cascaded theory is the comparison between single-distribution unequal variance and constrained bimodal. In all cases this provides better fit. What would happen in you just reported this in Table 5?

You're right that that's the comparison of interest. I think the main point here is that the model comparisons I present is the not the comparisons we need. We need to compare the models from the simplest to the most complicated model in pairs and not all models compared against the best fitting model.

**Comment by jensroes** [comment]
I've addressed most comments above. Some of the minor things (presentational stuff) I didn't address. Some other bits, especially Mark's comments that are unticked I have commented on above but also partly addressed in the manuscripts. That I didn't change something, doesn't mean I think its irrelevant to feel free to keep pushing :)

I've created a google docs folder with the text in individual documents. I think you should have editing access.
https://drive.google.com/drive/folders/1xLLgb3FKH9n4WRAN513QIyQtWC5im-yi?usp=drive_link

The full draft with figures and table is here: https://github.com/jensroes/prowrite-mixture-models/blob/main/report/manuscript.pdf

I'd like to submit this rather sooner than later. I'm a little bit under pressure to generate more first authored papers because promotions. Is there anything I can do to speed this up? Do you have time to look over this soon? Also I don't mind submitting even if it feels premature to get reviews. Mark knows my arguments around this :) of course we need to make sure that everything that is substantial is addressed. The only thing I'm worried about is the number of appendices. It's not really a problem but I do feel there are too many of them :)

**Comment by RConijn** [comment]
Shoot! I completely missed this comment! I have reserved some time today/tomorrow to work on this. 
From my previous reading I felt it was already OK to submit it rather soon - so let's make that happen.

**Comment by RConijn** [comment]
@jensroes given that we want to speed things up a bit, would you be OK with me rewriting some pieces of the discussion (with track changes), rather than giving overeraching feedback?

**Comment by jensroes** [comment]
Of course @RConijn :)

**Comment by RConijn** [comment]
@jensroes  Thanks! I've added my changes in the Word docs. Some more genera/larger comments:

1. You moved quite some of the model descriptions to the Appendix. I think this is a good choice. However, I feel that the part in the methods needs to be still understandable without having read the appendix - so some properties of the other models need to be there (see edits & comments)
2. Discussion for one of the fidings : "The first exception concern the GUNNEXP2 and SPL2 data sets: pauses were more likely to occur at before-sentence transitions compared to before-word transitions while in the remaining data sets hesitations were equally probable at before-word and before-sentence locations. " --> I wonder whether this might be related to the L2 proficiency for the SPL2 dataset: did you split between English (L1) and Spanish (L2) here? For the GUNNEXP2 I need more info (@Mark-Torrance : could you extend the methods on this dataset? any ideas for these findings?)
3. Considering your discussion about younger writers: In principle, we could make more of these comparisons (e.g., writing from sources, vs. writing without sources). Not sure if this adds much, and we might have needed an more diverse sample of datasets. 
This makes me wonder - the choice for the 6 datasets is mostly based on convenience. We did not directly control for specific artifacts between 2 datasets. This was obviously not the main aim, but would this make sense for FW? I can see this mixture modeling approach as a much more substantial way to compare two groups (e.g., for verifying quotes like Plakans (2008): "writers with the reading-to-write task eliciting a more interactive process for some writers and writing-only tasks requiring more initial and less online planning", which are only based on 40 TA protocols).
4. Considering the discussion about similar transition times for LIFT: I've asked Nina and Luuk, but they do not have a clear idea. I was thinking about structural isomorphism, but am not completely convinced of that either (also see other notes about Plantra).
5. Appendices: yes a lot. I think you could remove I, J, F, G. You already only refer to these in a footnote. For L, I wonder if a more general approach about this has been published before (see comment discussion), if so, you could also remove this one. Also, we should think about perhaps shortening the appendices. I have no idea what appendix B adds to the story (could be my lack of knowledge). I think H is useful, but then would need a bit morre text in the results section.

I know - this adds more questions than solves them, so I'm not sure if this really pushes things forward... Sorry
Even then, I feel it's almost there

**Comment by Mark-Torrance** [comment]
Looking at this now. What's the target journal?

**Comment by jensroes** [comment]
I'd aim for a general cognitive one like Cognition, JML, PBR but I'm open for suggestions.

**Comment by Mark-Torrance** [comment]
I've commented on just the abstract and introduction. 

https://docs.google.com/document/d/1grk-72S9xBfhgCo9gbyTB3Z97DdpUQ3u6OMZpqOG8bU/edit

To be honest, I think it needs quite a bit of work. I think a mainstream cognitive journal is a good (ambitious) target. But given this I think there's need for quite a bit of tightening and rearranging to make the introduction convincing. I'm pretty sure that all the right parts are there. They just need moving around and recasting into a clearer argument of the research. I've left you lots of comments to that effect.

Sorry to not be more positive. I know you want to get this off quickly, and it's taken me a long time to get round to looking at it.

**Comment by jensroes** [comment]
I've gone over all your comments now. I've left a few minor questions.

I'm tempted to leave all appendices in for now, although I agree we probably should remove some of them. Reason is, if I remove them or add them to a supplementary materials this makes work for me that a reviewer might overall because they feel it should be in the appendix. So I rather remove from the appendix as response to a reviewer. I don't think a long appendix is a reason to reject a paper.

What shall we do next? @RConijn do you want to add to the discussion (not sure what). @Mark-Torrance can you look at the other sections as well (if you find time at all)?

**Comment by jensroes** [comment]
@RConijn and @Mark-Torrance I'd like to get this submitted sooner than later. I think it would be useful to get reviewer feedback and as we aiming for a general cognitive journal, there are plenty and there is no problem with just submitting it somewhere else :)

Do you think we can get this submitted by the end of the month? I think the manuscript looks overall healthy and there are surely some things that can be changed but we might as well do that when we know what other people (reviewers) think. 

@Mark-Torrance what we definitely need is details for the GUNN experiment (see methods section; google drive folder).

**Comment by RConijn** [comment]
@jensroes thanks for the reminder. I've blocked some time on Friday to work on this

**Comment by Mark-Torrance** [comment]
I'm having a look at this now.

**Cascading**
Jens, remember the discussion about this at the seminar in Porto last summer. I had also had a longish chat with Olivia Afonso about this a few weeks ago. I think when most psycholinguists - particularly reading researchers - hear "cascading" they understand the same sort of model that we have, but the emphasis is different. The important part of cascading for them is that incomplete or incorrect representations become available before the one that eventually is processed for output. So, for example, the cloud / mashed potato image passes on activation from potato to lexical and then orthographic processes before CLOUD gets selected. So <p> might be active at the same time as <c>, <o>, <u>, <d>, slightly increasing the probability of typing <proud> rather than <cloud>. Not a very good example but you get the idea.

This is not the understanding that we want to communicate. Our point is just that the processing is semi parallel - in the sense described really well by Van Galen. I suggest we just talk about semi-parallel rather than cascading processes for this reason, explaining carefully in the text what we mean (which you may already do - I've only read the first sentence of the abstract so far).

**Comment by Mark-Torrance** [comment]
I've finished making suggestions for the introduction. I'll look at the rest tomorrow.

**Comment by Mark-Torrance** [comment]
"Present Study" section. Entirely up to you whether you include this. But I don't think it's odd having just one sub-heading - and lots of papers now do this. I find it useful, and I think reviewers will too.

**Comment by RConijn** [comment]
@jensroes I just added some very small comments. I don't have any further explanations for the results, maybe @Mark-Torrance can jump in here. I do think (but this might be my lack of knowledge about cog psy journals), that we might want to make more explicit how the approach could be applied in future work / which RQs to follow up on. Other than that I'm curious to hear what the reviewers think

**Comment by Mark-Torrance** [comment]
**Simulation** Not for action now, but picking this up from comments on the first draft - maybe something to consider for a re-submission if that proves necessary.

There are two questions that could be addressed by simulations. One is whether 2-mixture models will always provide better fit than a normal mixed effects model. That's what's in the appendix (I think - not checked yet). 

The other question is what keystrokes intervals look like, assuming semi-parallel, buffered processing ("cascading"), under different processing parameters. So directly simulate that two data-generating processes that we are hypothesising. This doesn't have to be anything like a full-blown model that include multiple parallel processes. Just a distribution for duration of upstream processes and another for motor processes. And a parameter for how many keystrokes the upstream processes scope over. Then model run of keystrokes, drawing from these distributions. Then describe and model resulting IKIs (under different parameter assumptions). (I don't know if that makes sense - but I hope you get the general idea.)

**Comment by Mark-Torrance** [comment]
> TODO: Mark, can you check / add information for C2L1, CATO, GUNNEXP2 as needed. For GUNNEXP2 I've added a provisional citation that probably needs changing;

I've made one or two edits in the text. For the table:

C2L1 - all good.
CATO - Did you use both dyslexic and non-dyslexic students? If so, make this clear in the table?
GUNNEXP2
- Should we change the name? Maybe GE2? Not a major thing, but it's a bit ... er ... unprofessional including an authors first name in the datafile name.
- English (NTU) students writing in English
- I've had a good look for age data and I think the conclusion is that we failed to collect this. So I think just leave blank.
- Task is argumentative essays.

**Comment by jensroes** [comment]
@Mark-Torrance in case you didn't see it, I responded to some of the comments you made here https://github.com/jensroes/prowrite-mixture-models/issues/5#issuecomment-1826779713 but I will add comments in the text too to rephrase what I mean.


> Do simulations from models of our datasets fit observed data?

There is an appendix with this information for all datasets and models.

> Do 2-mixture models will always provide better fit than a normal mixed effects model. So simulate data from a positively skewed single distribution and see if a two-distribution mixture model always provides better fit than a single log-normal distribution model.

Also in appendix. The result is that the mixture model does not provide a higher predictive performance ("fit")  if the data come from a single distribution process.

> What do keystrokes intervals look like, assuming semi-parallel, buffered processing ("cascading"), under different processing parameters

I understand what you mean, I think. All this is is simulating mixture model data with different $\theta$s and / or $\delta$s. I'm not sure why we would do that though :)

**Comment by jensroes** [comment]
> I've had a good look for age data and I think the conclusion is that we failed to collect this. So I think just leave blank.

If your sample is NTU students then at least the modal age is 19 (3rd year students and PG students rarely do experiments, you get some PhD students and RA, and literally everyone else is year 1 or year 2 students, mostly year 1)

**Comment by Mark-Torrance** [comment]
> I understand what you mean, I think. All this is is simulating mixture model data with different s and / or s. I'm not sure why we would do that though :)

I'm don't _think_ that it is as simple as that. This is what I tried to explain (not very well) in my comments about the unconstrained model. If it was just a case that beta came from one data generating process, and beta+delta from another. And a particular IKI observation is entirely explained by one or other of these, then you would be right. By which I mean, you have a particular transition and IKI is either associated with all of the motor planning for the next character, or it is associated with all of the non-motor planning. You're just swapping one for the other. But that's not the case. Because all you are actually observing is in the IKI is that part of the disfluent procsessing that could not be completed in parallel with previous keystrokes.

**Comment by jensroes** [comment]
How about this:

![mixdists](https://github.com/jensroes/prowrite-mixture-models/assets/26796124/464ee039-2493-4043-8119-32a7565d2ac8)

**Comment by jensroes** [comment]
Also, I think I will remove the unconstrained model but only when I move everything back into markdown.

I agree it works well but not without the actual parameter estimates and PIs. I think it is okay to have both plots in the results section.

**Comment by Mark-Torrance** [comment]
> I agree it works well but not without the actual parameter estimates and PIs. I think it is okay to have both plots in the results section.

With the caveat that you need a strong sentence in the text making clear that this shows the same data, but expressed directly in terms of model parameters. I still would argue that Figure 2 this is better in a table, though, whatever Gelman says :)  Visualisation is better done by (new) Figure 1. The problem with trying to visualise within word and before sentence in the same plot is that the important differences are an order of magnitude different. So they always disappear for within-word when you put them on the same plot.

New Figure 1 suggestions (I know the version above was just rough).
- flip rows and columns to avoid readers having to turn their laptops to one side (but also because this makes more sense - at least to me).
- Keep density scale the same across datasets within edge (I think this is important)
- Possibly omit all y axis labels and ticks. It's not an informative scale (to me, at least).

**Comment by jensroes** [comment]
[manuscript.pdf](https://github.com/jensroes/prowrite-mixture-models/files/15370269/manuscript.pdf)

This is the most recent version with updated figures.
